app/mailers/notify_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class NotifyMailer < GovukNotifyRails::Mailer
+  #
+  # Define methods as usual, and set the template and personalisation, if needed,
+  # then just use mail() as with any other ActionMailer, with the recipient email
+  #
+  def case_submitted(tribunal_case)
+    set_template('e64e9db9-d344-4041-a7df-135fbd39fa56')
```

Comment:
Can we have the template ID as a constant - either set in an env var or an initialiser?